### ISSUE TITLE
Take a stance on the "O" in OCaml

### ISSUE THREAD

**Issue by cies** [facebook/reason#574]
Sorry if this has been address: I could not find it anywhere.

What's Reason's stance on the "O" in OCaml? I cannot find any details on it from the public docs, and would like to know about it. Maybe a point to mention in the FAQ.

**Comment by SanderSpies** [comment]
It's still there.

Reason is an interface to OCaml and can transform OCaml code into Reason
code. This means it most support everything OCaml does.

**Comment by cies** [comment]
Ok. Does this need to be documented? Or is the translation between Reason and OCaml so straightforward that it needs no explanation?

btw: Feel free to close.

**Comment by chenglou** [comment]
Yeah the syntax is definitely there. But it's intentionally omitted. The object system in OCaml has its use but we've decided not to encourage it for now. Doesn't necessarily mean we don't like it; we actually got some neat ideas for its syntax but it's too experimental at the moment.

As an aside, I have these two snippets in my zshrc, on mac:
```
alias mlre="pbpaste | refmt -use-stdin true -parse ml -print re -is-interface-pp false | pbcopy"
alias reml="pbpaste | refmt -use-stdin true -parse re -print ml -is-interface-pp false | pbcopy"
```
Takes the code in your clipboard and turn it into the other syntax, and pastes that back into your clipboard. It's been a handy lightweight way of checking out what the reason syntax of some OCaml code is.

Hope that answers it! Closing for now.
